musig_wasm/src/lib.rs

```diff
@@ -0,0 +1,10 @@
+#[macro_use]
+extern crate lazy_static;
```

Comment:
>For some reason it doesn't work for me without this lines. I can investigate it, though I could be not the only one for whom this doesn't work, so I would leave it as is

In 2018 edition, you have to import macros just like any other items, e.g.:

[Pre-2018 approach](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=eb187f5c282ca48b81ff91874855bb3e) vs [2018 approach](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=a340d1067edf6471b8522563514d4f87).

Reply:
Fixed